lib/client.js

```diff
@@ -229,10 +243,28 @@ Client.prototype = {
   /// client.trigger('activation') // activating!
   /// ```
   trigger: function(name, data) {
-    if (!this._messageHandlers[name]) { return false; }
-    this._messageHandlers[name].forEach(function(handler) {
-      handler(data);
-    });
+    if (!this._messageHandlers[name] || repliesPending[name]) { return false; }
+    if (data && data.reply) {
```

Comment:
could potentially conflict with any user-defined `reply` property, perhaps move it up one level and allow a third parameter [here](https://github.com/zendesk/zendesk_app_framework_sdk/blob/6ecb1a2f78b8ea84e3680f3409b8ec7f60ae21c5/lib/client.js#L88)? Or as trigger is only ever called by the messageHandler itself for hooks, (we don't really want to support user-defined hooks do we), could potentially split this into a private method and leave trigger for the simple 'event' use case?